pages/common/cut.md

```diff
@@ -1,28 +1,24 @@
 # cut
 
 > Cut out fields from stdin or files.
-> More information: <https://www.gnu.org/software/coreutils/cut>.
+> More information: <https://manned.org/cut.1>.
```

Comment:
Again, the GNU coreutils link is preferable to a man page.

Reply:
When we have to use smth instead of manned.org? As a user I like unified manned.org interface. ;)

Reply:
I don't understand what you're saying here. What's `smth`?